sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -278,6 +278,8 @@ rename-mapping:
   GalleryArtifactVersionFullSource.virtualMachineId: -|arm-id
   SecurityPostureReference: ComputeSecurityPostureReference
   SecurityPostureReference.excludeExtensions: ExcludeExtensionNames
+  VirtualMachineScaleSet.properties.skuProfile : ComputeSkuProfile
```

Comment:
my comment is not asking to change the name of the property. We should change the name of the type of the property.
Sorry to be ambiguous, could you take a look again?
I added another comment to that place, and please revert this